index.html

```diff
@@ -0,0 +1,99 @@
+<head>
+    <meta charset="UTF-8">
+    <title>Richard Rabulan | Web Developer</title>
+    <link rel= "stylesheet" href= "normalize.css">
+    <link rel= "stylesheet" href= "style.css">
+</head>
+<a name="top"></a>
+<body>
+    <header class="navBar">
+        <div class="container">
+            <nav>
+                <ul>
+                    <li><a href="#home">HOME</a></li>
+                    <li><a href="#about">ABOUT ME</a></li>
+                    <li><a href="#feature">PORTFOLIO</a></li>
+                    <li><a href="#footer">CONTACT</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <main class="featureImage">
+        <div class="container">
+            <h1>RICHARD RABULAN</h1>
+            <p>Web Developer</p>
+            <button><a href="#feature">MORE</a></button>
+        </div>
+    </main>
+    <section class="portfolio">
+        <a name="feature"></a>
+        <div class="boxes">
+            <h3>PORTFOLIO</h3>
+            <div class="projectBox project1">
+                <h4>PROJECT 1</h4>
+            </div>
+            <div class="projectBox project2">
+                <h4>PROJECT 2</h4>
+            </div>
+            <div class="projectBox project3">
+                <h4>PROJECT 3</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+            <div class="projectBox project4">
+                <h4>PROJECT 4</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+            <div class="projectBox project5">
+                <h4>PROJECT 5</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+            <div class="projectBox project6">
+                <h4>PROJECT 6</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+        </div>
+    </section>
+    <section class="aboutMe">
+        <a name="about"></a>
+        <div class="container">
+            <h2>ABOUT ME</h2>
+            <p>My name is Richard and I am aspiring to become a full stack developer. I am currently studying at the Mayden Academy in Bath and have been recently certified as a Scrum Master.</p>
+            <p>In addition to my growing web development skills, I have a Bachelor of Arts degree in Architecture and have been working as an Architectural Technician for the last 3 years. During that time I have acquired skills that are similar in nature to being a developer as well as soft skills that makes me an excellent addition to a team. In my free time, I enjoy photography, cooking and creating digital artworks using architectural software.</p>
+        </div>
+    </section>
+    <footer class="footer">
+        <a name="footer"></a>
+        <section class="contact">
+                <h2>CONTACT ME</h2>
+                <p>02637 615387</p>
+                <p>emailme@email.com</p>
+        </section>
+        <div class="container">
+            <button><a href="#top">Back to Top</a></button>
+        </div>
+        <div class="container">
```

Comment:
I don't think you need this container. If you manage the margins you could do everything without it.